### PR TITLE
Update base images to add onbuild messages, balena-info, balena-idle commands

### LINKED ISSUE #612: Add custom idle command
A very common technique we use is to have a container idling. This could be either to troubleshoot an issue on an existing container or just as part of the development workflow when you don't want your container to exit but you are experimenting so don't have a proper `CMD` to keep it running.

The most common way of doing this is using this one-liner: 
- `CMD while : ; do echo "${MESSAGE=Idling...}"; sleep ${INTERVAL=600};`

See this project for more reference:
- https://github.com/balena-io-playground/balena-idling

It would be great to be able to achieve this with a custom command similar to [install_packages](https://github.com/balena-io-library/base-images/blob/master/balena-base-images/armv7hf/debian/stretch/run/Dockerfile#L26-L49). We could have for example `idle` or `balena_idle`, so that the one-liner turns into:
- `CMD balena_idle`

See this thread for a use case example: https://www.flowdock.com/app/rulemotion/balenalabs/threads/3YFfDbq9BJfcNlyzDpqR4mVJmo_

### PR DESCRIPTION
**Pull request by nghiant2710**
Connects to https://github.com/balena-io/contracts/pull/150
Fixes #612

### CODE REVIEW

File: scripts/assets/commands/balena-idle [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -e
+
+usage() {
+	cat <<EOUSAGE
+This script basically idles to keep the container running, while periodically printing to the logs that "Idling...".
```
Comment (shaunmulligan) [inline scripts/assets/commands/balena-idle:7]: Rephrase as " A small script that runs an idle process to keep the container running, while periodically printing the log: "Idling...""

File: scripts/assets/commands/balena-idle [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+Options:
+  -m	Message to print when ilding (Default to "Idling...").
```
Comment (shaunmulligan) [inline scripts/assets/commands/balena-idle:10]: perhaps we should add an option like `--silent` which doesn't log.

File: scripts/assets/commands/balena-idle (hunk 1 above)
Comment (nghiant2710) [reply scripts/assets/commands/balena-idle]: @shaunmulligan, updated.

File: scripts/assets/commands/balena-idle (hunk 2 above)
Comment (nghiant2710) [reply scripts/assets/commands/balena-idle]: updated.

**Pull request merged**